source/includes/cloudstack/_networks.md

```diff
@@ -232,7 +232,7 @@ curl -X DELETE \
    "https://cloudmc_endpoint/v1/services/compute-on/test_area/networks/9572d2ea-a60d-478a-a75e-8ed31f2641f1"
 ```
 
-<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networks/9572d2ea-a60d-478a-a75e-8ed31f2641f1</code>
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/networks/:id</code>
```

Comment:
The method should be `DELETE`

Reply:
good catch!

Reply:
Fixed, and found a few more small errors.